extensions-core/kinesis-indexing-service/pom.xml

```diff
@@ -20,90 +20,135 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+  <modelVersion>4.0.0</modelVersion>
```

Comment:
Oops changing the indent from 4 spaces to 2 spaces (like all of the other poms) borked the diff.  Mb

Reply:
Fwiw, adding `?w=1` to the PR URL (https://github.com/apache/incubator-druid/pull/6917/files?w=1) gets GitHub to show the diff in whitespace-doesn't-matter mode. In this case it makes it clearer what the real changes are.